fix mailserver config #3913

huanghaoyun opened the pull request:
fix #3913

fix mailserver config

dmitryn committed:
[#3352] Improve new message notifications

* Set custom sound
* Tap on notification opens chat
* Show message notifications when app in foreground

Signed-off-by: Dmitry Novotochinov <dmitry.novot@gmail.com>

goranjovic committed:
bug #5307 - showing new contract as recipient in contract creation transactions

Signed-off-by: Goran Jovic <goranjovic@gmail.com>

Commit:
Content refresh July 2018

Signed-off-by: Igor Mandrigin <i@mandrigin.ru>

goranjovic committed:
refactor - extracted all model functions from profile events

Signed-off-by: Goran Jovic <goranjovic@gmail.com>

goranjovic committed:
refactor - extracted all model functions from accounts events

Signed-off-by: Goran Jovic <goranjovic@gmail.com>

huanghaoyun committed:
fix mailserver config #3913

fix format

ghost commented:
Hey @huanghaoyun, and thank you so much for making your first pull request in status-react! :heart: Please help us make your experience better by filling out this brief questionnaire https://goo.gl/forms/uWqNcVpVz7OIopXg2

<blockquote><img src="https://lh5.googleusercontent.com/wpBxRKA0YohehKsTaptQsaYJNVcNA9m2IOLvxXpSMcamdM8e4q7gyYBQ_yRVvNH-Ez8=w1200-h630-p" width="48" align="right"><div><img src="//ssl.gstatic.com/docs/forms/device_home/android_192.png" height="14"> Google Docs</div><div><strong><a href="https://docs.google.com/forms/d/e/1FAIpQLSewRMgMD6FLe9yO7oCipVn7YTn1HboJc4ZDJ_Va_BsDAemQwQ/viewform?usp=send_form&usp=embed_facebook">Status 1st commit dev survey</a></strong></div><div>First of all thank you so much for taking the time to make your first foray into the world of Status. We want to make sure that your experience with building the future of the decentralized web with us is not only enjoyable but gives you real purpose.

 Please check out our wiki at https://wiki.status.im and our Dev forum at https://discuss.status.im for more in depth longform discussion -- You can also join our Riot at https://chat.status.im which has all of our developers to answer any pressing questions you may have.</div></blockquote>

antdanchenko committed:
added ability to set default network

Signed-off-by: Anton Danchenko <ant.danchenko@gmail.com>

yenda commented:
@huanghaoyun thanks for contributing ! It looks like there is a conflict with your PR, could you resolve it ?
Also the issue also includes disconnecting from the previous mailserver, could you add that to the PR ? Thanks

jakubgs committed:
move archive.plist settings to the repo

Signed-off-by: Jakub Sokołowski <jakub@status.im>

jakubgs committed:
fix one more path for ios/archive-develop.plist

mandrigin committed:
Bump `develop` version to 0.9.26 since we are submitting 0.9.25 as a hotfix.

Signed-off-by: Igor Mandrigin <i@mandrigin.ru>

yevh-berdnyk committed:
Fixed public key and chat name inputs and amount for tokens

Signed-off-by: yevh-berdnyk <ie.berdnyk@gmail.com>

jakubgs committed:
fix typo in ci/Jenkinsfile.desktopbuild

yevh-berdnyk committed:
Changed run name and TestRail report for PRs

Signed-off-by: yevh-berdnyk <ie.berdnyk@gmail.com>

jakubgs committed:
fix removal of StatusAppFiles

751895405 commented:
I will handle conflict issue and also I will check the previous mailserver connection state

jakubgs committed:
fix git tag push credentials, needs to use token instead of pass

Signed-off-by: Dmitry Novotochinov <dmitry.novot@gmail.com>

yevh-berdnyk committed:
Updated captions in tests

Signed-off-by: yevh-berdnyk <ie.berdnyk@gmail.com>

jakubgs committed:
cleanup jenkinsfile and prepare for builds on new linux slaves

jakubgs committed:
fix which job is run to update nightly URLs

Signed-off-by: Dmitry Novotochinov <dmitry.novot@gmail.com>

maxhora committed:
Enable scroll-view rendering optimization for array of children #5012

jakubgs committed:
fix variable name for AppImage url

huanghaoyun commented:
@yenda there is no connection between mobile client and mailserver ,I  think i should check status-go code to see if there is a method for remove peers .

dmitryn committed:
Update cljs-rn to new version

:watch task use new defaults that don't run app and metro bundler by
default

Signed-off-by: Dmitry Novotochinov <dmitry.novot@gmail.com>

mandrigin committed:
[#4823] AppStoreConnect: Mark our iOS app as encryption-compliant.

To avoid specifying that we are compliant manually for every build,
we put these values into build setting of the iOS app.
More information on how to do that is here: https://stackoverflow.com/a/36028077

Signed-off-by: Igor Mandrigin <i@mandrigin.ru>

jakubgs committed:
use password for the right host, run before xcodebuild

Signed-off-by: Igor Mandrigin <i@mandrigin.ru>

huanghaoyun committed:
fixes mailserver config
